build/vsts-helpers.ps1

```diff
@@ -1,6 +1,5 @@
-﻿function WriteHeader
+function WriteHeader
```

Comment:
Do we want to rename this function to `Write-Header` instead, to be compliant with powershell best practices?

Reply:
Of course, because this is a Advanced function, it has most of the features of a cmdlet.

This function is only used internally, the frequency of use is very low, and the readability of the name is not a big problem, so it can be changed or not. You can choose according to your mood.

Reply:
Fuck it. It's internal so it doesn't really matter at the end of the day. It can be kept as is.